apache2/conf-enabled/sbf-custom-mime.conf

```diff
@@ -0,0 +1 @@
+../conf-available/sbf-custom-mime.conf
```

Comment:
This repo was _initially_ intended to be public - similar to the default apache images on the dockerhub with just the modules that we need. This repo and the Quay repo are public. Things like this are making it SBF-specific.

I suppose I'm ok with it. However we should make the repos private then. Also, if we do that we probably don't need to have a separate dockerfile in our repo that  FROM's this one, we can probably just combine it all into one.